### PR TITLE
chore(rebase): Rebase the v6 branch

### LINKED ISSUE #10127: rebase v6
**Describe the enhancement or change**
A clear and concise description of the request. What is the expected outcome?

**Is this request originating from a Red Hat product team? If so, which ones and is there any sort of deadline for this enhancement?**

**Any other information?**

### PR DESCRIPTION
**Pull request by tlabaj**
**What**: Closes #10127

### CODE REVIEW

**Review by thatblindgeye — APPROVED** [review]

**Review by wise-king-sullyman — COMMENTED** [review]
One comment that I think we might want to address before merging this, but other than that looks good as far as I can tell

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -329,28 +329,14 @@ jobs:
...
-        with:
-          path: |
-            packages/*/dist
-            packages/*/next
-            packages/*/deprecated
-            packages/*/components
-            packages/react-styles/css
-            packages/react-core/layouts
-            packages/react-core/helpers
-          key: ${{ runner.os }}-dist-14-${{ secrets.CACHE_VERSION }}-${{ hashFiles('yarn.lock', 'package.json', 'packages/*/*', '!packages/*/dist', '!packages/*/node_modules') }}
```
[the remark this answers is not in the document]
Comment (tlabaj) [reply .github/workflows/release.yml:350]: Is that outside scope of this PR?  Don't we want to do that in v5 as well?

[the remark this answers is not in the document]
Comment (wise-king-sullyman) [reply .github/workflows/release.yml:350]: We would want to do it in v5 as well, but I don't think we would really want to bring this change into v6 in the first place. Not a blocker for me though.

**Review by wise-king-sullyman — APPROVED** [review]

File: .github/workflows/release.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (wise-king-sullyman) [reply .github/workflows/release.yml]: Followup issue to readd to main #10146

**Review by wise-king-sullyman — APPROVED** [review]

**Pull request merged**